bundle/manifests/kepler-operator.clusterserviceversion.yaml

```diff
@@ -347,9 +333,11 @@ spec:
     name: Parul Singh
   - email: hchen@redhat.com
     name: Huamin Chen
+  - email: sthaha@redhat.com
+    name: Sunil Thaha
   maturity: alpha
   provider:
     name: Kepler Operator Contributors
     url: https://sustainable-computing.io/
-  version: 0.4.1
   replaces: kepler-operator.v0.4.0
```

Comment:
`replaces: kepler-operator.v0.4.1`

Reply:
I did some more reading. We need to update the `replaces` because we use `replaces-mode` instead of `semver-mode` in the ci.yaml.  [relaces-mode](https://redhat-openshift-ecosystem.github.io/community-operators-prod/operator-ci-yaml/#replaces-mode) is the default so if we switch to `semver-mode` updating the `replaces` will not be required.

Reply:
Although this is true, I believe it is best to stick to replaces-mode until we know for use that with `semver` API breakages are okay or have a clear idea on how to specify it in the bundle manifests.

Reply:
> `replaces: kepler-operator.v0.4.1` `replaces` is for replacing the previous csv version.

@vprashar2929 If you can fix the `version` and `replaces` property we can close this PR.
Thanks.